lib/services/ios-project-service.ts

```diff
@@ -219,6 +229,13 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
     <string>${options.teamID}</string>
 `;
 		}
+		if (options && options.provision) {
+			plistTemplate += `    <key>provisioningProfiles</key>
```

Comment:
A separate service/helper function could be extracted for getting the `exportOptions.plist` file and be reused here and [here](https://github.com/NativeScript/nativescript-cli/pull/3070/files#diff-b673f137deb4afccccc0bd5f1572c2a0R280).
This could be done in another PR though